src/main/java/com/quizapp/enterprise/persistence/GameTracker.java

```diff
@@ -101,14 +101,11 @@ public Player getPlayer(String gameCode, String username){
 
     public ArrayList<Player> getLeaderboard(String gameId){
         ArrayList<Player> players = getGameByCode(gameId).getPlayers();
-         Collections.sort(players, new Comparator<Player>() {
-            @Override
-            public int compare(Player o1, Player o2) {
-                if(o1.getTotalPoints() > o2.getTotalPoints()){
-                    return -1;
-                }else {
-                    return 1;
-                }
+        Collections.sort(players, (o1, o2) -> {
```

Comment:
Lamba functions are more memory efficient and use less code to implement

https://www.w3schools.com/java/java_lambda.asp

Reply:
Huh.  Nice refactor.